core/backend/src/IModelDb.ts

```diff
@@ -365,7 +365,22 @@ export abstract class IModelDb extends IModel {
       });
     }
   }
-
+  /**
+   * Attach a iModel file to this connection and load and register its schemas.
+   * @param fileName IModel file name
+   * @param alias identifier for the attached file. This identifer is used a tablespace executing ECSQL queries.
```

Comment:
Same as for ECDb regarding "a tablespace".

Reply:
fixed